### ISSUE TITLE
Feature request: Right click to swap tools in toolbelt

### ISSUE THREAD

**Issue by Nuaja** [Lizard-Of-Oz/Inventorio#134]
We can already shift click a tool to put it in the toolbelt, but I thought of a feature to make that even more useful. It could be right click instead, and it would be also able to replace the tool that's currently in the toolbelt. So for example, if I want to swap my Silk pick and my Fortune pick, I'd just have to right click my Silk pick in my inventory to replace the Fortune pick that's in my toolbelt.

**Comment by Lizard-Of-Oz** [comment]
Certain tools (e.g. axes, hoes, shears) already have a right-click functionality. This is even more expected with modded tools, and the latter makes it impossible to universally account for.

**Comment by Nuaja** [comment]
> Certain tools (e.g. axes, hoes, shears) already have a right-click functionality. This is even more expected with modded tools, which makes it impossible to universally account for.

This is not what I meant. What I'm talking about is right clicking on the tool in the inventory

**Comment by Lizard-Of-Oz** [comment]
Right-click doesn't have unique functionality in Minecraft on itself. It'd be odd to make an exception for tools,  giving that people aren't expecting that to do anything different anyway.
